config/default.yaml

```diff
@@ -36,4 +36,7 @@ redis:
     # Password to connect to redis cluster
     # password: null
     # Host of redis cluster
-    port: 6379
+    port: "6379"
```

Comment:
Not sure if this makes any difference but in the [docs](https://github.com/taskrabbit/node-resque/blob/master/README.md) for `node-resque` the examples all show `port` being a number and not string